tools/provision

```diff
@@ -51,8 +57,10 @@ the Python version this command is executed with."""
     # the venv's Python interpreter. `--prefix venv_dir` ensures that all modules are installed
     # in the right place.
     def install_dependencies(requirements_filename):
-        if subprocess.call([os.path.join(venv_dir, venv_exec_dir, 'pip'),
-                           'install', '--prefix', venv_dir, '-r', os.path.join(base_dir, requirements_filename)]):
+        pip_path = os.path.join(venv_dir, venv_exec_dir, 'pip')
+        subprocess.call([pip_path, 'install', 'pip>=9.0'])
```

Comment:
Instead of making another call, could we add `pip>=9.0` to `requirements.txt`?

Reply:
The `pip` must be updated first so that the >=9.0 version of it can take effect.